setup.cfg

```diff
@@ -163,6 +163,10 @@ filterwarnings =
     # Currently helio makes unverified requests - this filter should be removed when
     # https://github.com/sunpy/sunpy/issues/4401 is fixed
     ignore:Unverified HTTPS request
+    # Seems there is a 3.9 deprecation warning inside ipykernel
```

Comment:
Please add a comment that links to an upstream issue on this, and make the ignores more specific by adding their message (or at least part of it)